cypress/support/fragments/data_import/action_profiles/newActionProfile.js

```diff
@@ -0,0 +1,45 @@
+import { TextField, Button, Select } from '../../../../../interactors';
+import { getLongDelay } from '../../../utils/cypressTools';
+import ModalSelectMappingProfile from './modalSelectMappingProfile';
+
+const profileName = {
+  instanceName: 'Instance',
+  holdingsName: 'Holdings',
+  itemName: 'Item',
+};
+
+const action = {
+  create: 'Create (all record types)',
+};
+
+const folioRecordTypeValue = {
```

Comment:
crossed with profileName. May be you can use this object as name too and remove profileName after this from your code